client/src/store/slices/login/index.ts

```diff
@@ -45,8 +46,12 @@ export const googleLogin = createAsyncThunk(
       const googleToken = await api.loginGoogle(accessToken);
       return await api.login(googleToken.accessToken);
     } catch (err) {
-      console.error(err.response.data);
-      return rejectWithValue(err.response.data);
+      if (axios.isAxiosError(err)) {
```

Comment:
This if-else statement is re-used a few times in this file as well as at client/src/store/slices/mentor/index.ts, so it should be written as a helper function (in /client/src/helpers.ts) and imported wherever it's needed